Functions/ImageProcessing/xASL_im_ASLSubtractionAveraging.m

```diff
@@ -182,8 +182,13 @@
     % Load ASL4D.nii
     ASL4Dnii = xASL_io_ReadNifti(path_ASL4D);
     sizeASL4D = size(ASL4Dnii.dat);
-    nVolumes = sizeASL4D(4);
-    nDims = length(sizeASL4D);
+	nDims = length(sizeASL4D);
```

Comment:
You can do size(IM,4) which will by default give 1 if nDims<4, for shorter code

Reply:
Sure. Done in 467dc9c8